druid-mr/src/main/java/com/yahoo/druid/hadoop/DruidHelper.java

```diff
@@ -83,7 +84,19 @@ public String apply(TimelineObjectHolder<String, DataSegment> input)
         int responseCode = conn.getResponseCode();
         if(responseCode == 200) {
           ObjectMapper mapper = new DefaultObjectMapper();
-          Map<String,Object> obj = 
```

Comment:
Instead, can you do
ObjectMapper mapper = DruidInitialization.getInstance().getObjectMapper() and add necessary modules to that one if needed.